UnityProject/Assets/Scripts/Health/RespiratoryHealth/RespiratorySystem.cs

```diff
@@ -119,7 +119,7 @@ private bool Breathe(IGasMixContainer node)
 		if (oxygenUsed > 0)
 		{
 			breathGasMix.RemoveGas(Gas.Oxygen, oxygenUsed);
-			breathGasMix.AddGas(Gas.CarbonDioxide, oxygenUsed);
+			node.GasMix.AddGas(Gas.CarbonDioxide, oxygenUsed);
```

Comment:
Any time you change the gasmix on a node I am pretty sure you need to call MetaDataLayer.UpdateSystemsAt(position), otherwise atmos won't reliably detect the change. See how it is done in GasContainer.CheckRelease()

Reply:
Fixed.